test/smokes/pmd_cpd/expectations.rb

```diff
@@ -0,0 +1,378 @@
+      id: "069e8856ac66563b0c6e50b615209face9958fa7",
+      message: "Code duplications found (2 occurrences).",
+      links: [],
+      object: {
+        lines: 28,
+        tokens: 111,
+        files: [
+          {
+            id: "e8eee93de21372ef3086ae97d2d0e998f15e96e7",
+            path: "app.java",
+            start_line: 1,
+            start_column: 1,
+            end_line: 28,
+            end_column: 1
+          },
+          {
+            id: "069e8856ac66563b0c6e50b615209face9958fa7",
+            path: "app.java",
+            start_line: 31,
+            start_column: 1,
+            end_line: 58,
+            end_column: 1
+          }
+        ],
+        codefragment: %[public class Main {
+  public static void main(String[] args) {
+    int day = 4;
+    switch (day) {
+      case 1:
+        System.out.println("Monday");
+        break;
+      case 2:
+        System.out.println("Tuesday");
+        break;
+      case 3:
+        System.out.println("Wednesday");
+        break;
+      case 4:
+        System.out.println("Thursday");
+        break;
+      case 5:
+        System.out.println("Friday");
+        break;
+      case 6:
+        System.out.println("Saturday");
+        break;
+      case 7:
+        System.out.println("Sunday");
+        break;
+    }
+  }
+}]
+      },
+      git_blame_info: nil
+    },
+    {
+      path: "app.java",
+      location: {
+        start_line: 1,
+        start_column: 1,
+        end_line: 28,
+        end_column: 1
+      },
+      id: "e8eee93de21372ef3086ae97d2d0e998f15e96e7",
+      message: "Code duplications found (2 occurrences).",
+      links: [],
+      object: {
+        lines: 28,
+        tokens: 111,
+        files: [
+          {
+            id: "e8eee93de21372ef3086ae97d2d0e998f15e96e7",
+            path: "app.java",
+            start_line: 1,
+            start_column: 1,
+            end_line: 28,
+            end_column: 1
+          },
+          {
+            id: "069e8856ac66563b0c6e50b615209face9958fa7",
+            path: "app.java",
+            start_line: 31,
+            start_column: 1,
+            end_line: 58,
+            end_column: 1
+          }
+        ],
+        codefragment: %[public class Main {
+  public static void main(String[] args) {
+    int day = 4;
+    switch (day) {
+      case 1:
+        System.out.println("Monday");
+        break;
+      case 2:
+        System.out.println("Tuesday");
+        break;
+      case 3:
+        System.out.println("Wednesday");
+        break;
+      case 4:
+        System.out.println("Thursday");
+        break;
+      case 5:
+        System.out.println("Friday");
+        break;
+      case 6:
+        System.out.println("Saturday");
+        break;
+      case 7:
+        System.out.println("Sunday");
+        break;
+    }
+  }
+}]
+      },
+      git_blame_info: nil
+    }
+  ],
+  analyzer: { name: "PMD CPD", version: "6.23.0" }
+)
+
+s.add_test(
+  "broken_sideci_yml",
```

Comment:
[suggestion] `sideci.yml` is old, so please use `sider.yml` instead. 🙏 


```suggestion
  "broken_sider_yml",
```

Reply:
peace of cake.
Fixed. commit 50308ffeecbc22479ce5bdb1e8f3ac065b4c0530